helix-term/src/ui/statusline.rs

```diff
@@ -347,10 +347,16 @@ fn render_selections<F>(context: &mut RenderContext, write: F)
 where
     F: Fn(&mut RenderContext, String, Option<Style>) + Copy,
 {
-    let count = context.doc.selection(context.view.id).len();
+    let selection = context.doc.selection(context.view.id);
+    let count = selection.len();
     write(
         context,
-        format!(" {} sel{} ", count, if count == 1 { "" } else { "s" }),
+        format!(
+            "{}/{} sel{} ",
+            selection.primary_index() + 1,
+            count,
+            if count == 1 { "" } else { "s" }
+        ),
```

Comment:
```suggestion
        if count == 1 {
            "1 sel ".to_owned()
        } else {
            format!("{}/{count} sels ", selection.primary_index() + 1)
        },
```

The only other preference I have for this is to keep the display as `1 sel` until there are multiple selections. With only one selection it would always be `1/1 sel` otherwise which just takes up space